Formula/umlet.rb

```diff
@@ -0,0 +1,50 @@
+class Umlet < Formula
+  desc "This UML tool aimed at providing a fast way of creating UML diagrams."
+  homepage "http://www.umlet.com/"
+  url "http://www.umlet.com/umlet_14_2/umlet-standalone-14.2.zip"
+  sha256 "dc09538d04cb899218f3bdfdb5205f60359cff68cb1428d1228c6d91743d8cb9"
+
+  bottle :unneeded
+
+  keg_only ""
+
+  depends_on :java => "1.6+"
+
+  def install
+    # Remove unnecessary files
+    rm Dir["*.{desktop,exe}"]
+
+    # Move all subdirectories and files but without lib/
+    prefix.install Dir["*"]
+
+    # # Move all subdirectories and files but without lib/
+    # 'lib/' is hard-coded in umlet.jar!META-INF/MANIFEST
+    # # Moving to 'libexec/' doesn't work :-(
+    # prefix.install Dir["*"] - ["lib"]
+    # # Move content of lib/ into libexec/
+    # libexec.install Dir["lib/*"]
+    # java_classpath = Dir[prefix/"**/*.jar"].join(":")
+
+    # Replace 'java' command in shell script
+    inreplace "#{prefix}/umlet.sh", " java ", " ${JAVA_HOME}/bin/java "
+
+    # Deactivate use of 'programDir'
+    inreplace "#{prefix}/umlet.sh", "programDir=", "#programDir="
+
+    # Replace '${programDir}'
+    inreplace "#{prefix}/umlet.sh", "${programDir}/umlet.jar", "#{prefix}/umlet.jar"
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patches are needed in a comment in the formula file. Thanks!